test/acceptance/features/workloadResourceBinding.feature

```diff
@@ -0,0 +1,468 @@
+            """
+
+    Scenario: Specify container path using Workload Resource Mappings in custom resource
+        Given The Secret is present
+            """
+            apiVersion: v1
+            kind: Secret
+            metadata:
+                name: $scenario_id-secret
+            stringData:
+                username: AzureDiamond
+                password: hunter2
+            """
+        And The Custom Resource is present
+            """
+            apiVersion: stable.example.com/v1
+            kind: Backend
+            metadata:
+                name: $scenario_id-backend
+                annotations:
+                    service.binding: path={.status.data.dbCredentials},objectType=Secret,elementType=map
+            status:
+                data:
+                    dbCredentials: $scenario_id-secret
+            """
+        And The Workload Resource Mapping is present
+            """
+            apiVersion: servicebinding.io/v1alpha3
+            kind: ClusterWorkloadResourceMapping
+            metadata:
+                name: appconfig.stable.example.com
+            spec:
+                versions:
+                  - version: "*"
+                    containers:
+                      - path: .spec.spec.containers[*]
+                      - path: .spec.spec.initContainers[*]
+                    volumes: .spec.spec.volumes
+            """
+        And The Custom Resource is present
+            """
+            apiVersion: "stable.example.com/v1"
+            kind: AppConfig
+            metadata:
+                name: $scenario_id-appconfig
+            spec:
+                uri: "some uri"
+                Command: "some command"
+                image: my-image
+                spec:
+                    containers:
+                    - name: hello-world
+                      # Image from dockerhub, This is the import path for the Go binary to build and run.
+                      image: yusufkaratoprak/kubernetes-gosample:latest
+                      ports:
+                      - containerPort: 8090
+            """
+        When Service Binding is applied
+            """
+            apiVersion: servicebinding.io/v1alpha3
+            kind: ServiceBinding
+            metadata:
+                name: $scenario_id-binding
+            spec:
+                service:
+                    name: $scenario_id-backend
+                    kind: Backend
+                    apiVersion: stable.example.com/v1
+                workload:
+                    apiVersion: stable.example.com/v1
+                    kind: appconfigs
+                    name: $scenario_id-appconfig
+            """
+        Then Service Binding is ready
+        And The application env var "SERVICE_BINDING_ROOT" has value "/bindings"
+        And Content of file "/bindings/$scenario_id-binding/username" in application pod is
+            """
+            AzureDiamond
+            """
```

Comment:
Where is the pod getting created? The `AppConfig` is a custom resource, and there is no controller to create a pod. Probably we need to make the assertion in some other way.

Reply:
When I originally wrote this, I definitely thought `AppConfig` resources made pods.  I've since rewritten this so that the assertion makes more sense in this context.